.gitignore

```diff
@@ -0,0 +1,4 @@
+*.egg-info
+__pycache__/
+.coverage*
+dist/
```

Comment:
These files are not specific to advent-of-code-data, so should go in a user (global) .gitignore file not a local one.

Reply:
I disagree, but it's out of scope for this PR regardless. I'll revert this.